package.json

```diff
@@ -3,14 +3,19 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@styled-icons/bootstrap": "^10.19.0",
+    "@styled-icons/boxicons-regular": "^10.18.0",
+    "@styled-icons/material": "^10.18.0",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^10.4.9",
     "@testing-library/user-event": "^12.1.3",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.3"
+    "react-scripts": "3.4.3",
+    "react-test-renderer": "^17.0.1",
```

Comment:
Unfortunately, the code coverage report is not working properly
<img width="504" alt="Screen Shot 2020-12-16 at 21 45 55" src="https://user-images.githubusercontent.com/6569533/102446175-68517c80-3fea-11eb-98ae-aeedb8fd937d.png">